crates/router/src/connector/adyen/transformers.rs

```diff
@@ -3325,14 +3297,22 @@ pub fn get_qr_metadata(
     let image_data_url = Url::parse(image_data.data.as_str())
         .ok()
         .ok_or(errors::ConnectorError::ResponseHandlingFailed)?;
+    let qr_code_url = response
+        .action
+        .qr_code_url
+        .clone()
+        .ok_or(errors::ConnectorError::ResponseHandlingFailed)?;
```

Comment:
Why we are throwing error incase of None? This is an optional field from connector